diff --git a/apis/v1/queue/oas_client_gen.go b/apis/v1/queue/oas_client_gen.go
index 6ab5c8d..27d14d4 100644
--- a/apis/v1/queue/oas_client_gen.go
+++ b/apis/v1/queue/oas_client_gen.go
@@ -658,7 +658,7 @@ func (c *Client) sendGetQueues(ctx context.Context) (res GetQueuesRes, err error
 
 	u := uri.Clone(c.requestURL(ctx))
 	var pathParts [1]string
-	pathParts[0] = "/commonserviceitem"
+	pathParts[0] = `/commonserviceitem?{"Filter":{"Provider.Class":"simplemq"}}` // NOTE: ここだけOpenAPIで表現できず手動で書き加えている
 	uri.AddPathParts(u, pathParts[:]...)
 
 	r, err := ht.NewRequest(ctx, "GET", u)
